zqd/space/space.go

```diff
@@ -422,6 +422,10 @@ type info struct {
 	MaxTime nano.Ts `json:"max_time"`
 }
 
+func (i info) Span() nano.Span {
+	return nano.NewSpanTs(i.MinTime, i.MaxTime)
+}
+
 // UnsetTimes nils out the cached time range value for the space.
 // XXX For right now this simply deletes the info file as nothing else is stored
 // there. When we get to brimsec/zq#541 the time range should be represented as
```

Comment:
Looks like this comment should be updated if this isn't being done in #541

Reply:
Yep